src/main/java/chatkaki/ChatKaki.java

```diff
@@ -0,0 +1,45 @@
+package chatkaki;
+
+import java.util.Scanner;
+
+import chatkaki.commands.Command;
+
+/**
+ * Represents the ChatKaki chatbot.
+ */
+public class ChatKaki {
+
+    /**
+     * Starts the chat service.
+     */
+    public static void startChatService() {
```

Comment:
The startChatService() function does multiple things: it greets the user, handles user input, and manages command execution. Maybe you can consider breaking it down into separate methods for better readability. However, this allows the code to be concise which is good as well